ocaml/xapi/test_vdi_cbt.ml

```diff
@@ -190,11 +191,33 @@ let test_get_nbd_info =
   ; "test_disallowed_for_cbt_metadata_vdi" >:: test_disallowed_for_cbt_metadata_vdi
   ]
 
+let test_allowed_operations_updated_when_necessary () =
+  let __context = Test_common.make_test_database () in
+  let host = Helpers.get_localhost ~__context in
+  let sR = Test_common.make_sr ~__context () in
+	(* This SM instance has CBT capabilities by default *)
+  let _: _ API.Ref.t = Test_common.make_sm ~__context () in
+  let _: _ API.Ref.t = Test_common.make_pbd ~__context ~host ~sR () in
+  let self = Test_common.make_vdi ~__context ~sR ~_type:`user ~cbt_enabled:true ~is_a_snapshot:true ~managed:true () in
+  register_smapiv2_server ~vdi_data_destroy:(fun _ ~dbg ~sr ~vdi -> ()) (Db.SR.get_uuid ~__context ~self:sR);
+
+  let assert_allowed_operations msg check =
+    OUnit.assert_bool msg (check (Db.VDI.get_allowed_operations ~__context ~self))
+  in
+  (* Populate the allowed_operations list after creating the VDI *)
+  Xapi_vdi.update_allowed_operations ~__context ~self;
+  assert_allowed_operations "contains `copy for a newly-created VDI" (fun ops -> List.mem `copy ops);
+	let (rpc, session_id) = Test_common.make_client_params ~__context in
```

Comment:
Why are these three lines indented further?

Reply:
good catch, I'll correct it